packages/styleguide/.storybook/main.ts

```diff
@@ -4,8 +4,8 @@ const getStories = require('./getStories');
 
 // https://github.com/storybookjs/storybook/issues/12262#issuecomment-681953346
 // make a shallow copy of an object, rejecting keys that match /emotion/
-function emotionless(object) {
-  let result = {};
+function emotionless(object: any) {
+  let result = {} as any;
```

Comment:
Soon, we're going to lint against `any`...

```suggestion
function emotionless(object: Record<string, unknown>) {
  let result = {} as Record<string, unknown>;
```